numpy/_core/memmap.py

```diff
@@ -162,48 +162,48 @@ class memmap(ndarray):
 
     Load the memmap and verify data was stored:
 
-    >>> newfp = np.memmap(filename, dtype='float32', mode='r', shape=(3,4))
-    >>> newfp
+    >>> fp_new = np.memmap(filename, dtype='float32', mode='r', shape=(3,4))
+    >>> fp_new
```

Comment:
Did codespell do this? It seems unnecessary

Reply:
nope, I did that in 639693ee4fa82173f5444e76cfec47704d655ea6 :

```
commit 639693ee4fa82173f5444e76cfec47704d655ea6
Author: Yaroslav Halchenko <debian@onerussian.com>
Date:   Fri Apr 19 10:02:05 2024 -0400

    DOC: adjust variables in example to be fp_{kind} instead of {k}fp and fp{k}
    
    I think it makes them more consistent (always a suffix) and also more
    readable since more descriptive of the taken action.
    
    This is triggered to avoid "looks-like-a-typo" hits  in codespell for fpr and likely
    others
```